xbmc/interfaces/json-rpc/schema/types.json

```diff
@@ -820,7 +820,7 @@
   "PVR.Fields.Channel": {
     "extends": "Item.Fields.Base",
     "items": { "type": "string",
-      "enum": [ "thumbnail", "channeltype", "hidden", "locked", "channel", "lastplayed", "broadcastnow", "broadcastnext" ]
+      "enum": [ "uniqueid", "thumbnail", "channeltype", "hidden", "locked", "channel", "lastplayed", "broadcastnow", "broadcastnext"]
```

Comment:
New fields must be added at the end, as JSON specs says (and Montellese), to assure retro compatibility.